<?xml version="1.0" encoding="UTF-8"?>

<ui version="4.0">
<author>
SPDX-FileCopyrightText: 2019-2023 Mattia Basaglia <dev@dragon.best>
SPDX-FileCopyrightText: 2024 Julius KÃ¼nzel <julius.kuenzel@kde.org>
SPDX-License-Identifier: GPL-3.0-or-later
</author>
 <class>glaxnimate::gui::AboutEnvironmentDialog</class>
 <widget class="QDialog" name="glaxnimate::gui::AboutEnvironmentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About Environment</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Paths</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>463</width>
            <height>438</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>User Data Directory:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="line_user_data">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton">
               <property name="toolTip">
                <string>Open</string>
               </property>
               <property name="text">
                <string>Open</string>
               </property>
               <property name="icon">
                <iconset theme="folder-open"/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Settings File:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLineEdit" name="line_settings">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_2">
               <property name="toolTip">
                <string>Open</string>
               </property>
               <property name="text">
                <string>Open</string>
               </property>
               <property name="icon">
                <iconset theme="document-open"/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Backup Directory:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLineEdit" name="line_backup">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_3">
               <property name="toolTip">
                <string>Open</string>
               </property>
               <property name="text">
                <string>Open</string>
               </property>
               <property name="icon">
                <iconset theme="folder-open"/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Data Directories:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="view_data">
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
             </property>
             <property name="textElideMode">
              <enum>Qt::TextElideMode::ElideMiddle</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Icon Theme Paths:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="view_icons">
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
             </property>
             <property name="textElideMode">
              <enum>Qt::TextElideMode::ElideMiddle</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Formats</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea_3">
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>483</width>
            <height>458</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Input Formats</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="table_formats_input">
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
             </property>
             <property name="textElideMode">
              <enum>Qt::TextElideMode::ElideNone</enum>
             </property>
             <property name="horizontalScrollMode">
              <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
             </property>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>ID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Extensions</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Output Formats </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="table_formats_output">
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
             </property>
             <property name="textElideMode">
              <enum>Qt::TextElideMode::ElideNone</enum>
             </property>
             <property name="horizontalScrollMode">
              <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
             </property>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>ID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Extensions</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>glaxnimate::gui::AboutEnvironmentDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>view_data</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>glaxnimate::gui::AboutEnvironmentDialog</receiver>
   <slot>dir_open(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>617</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>view_icons</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>glaxnimate::gui::AboutEnvironmentDialog</receiver>
   <slot>dir_open(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>563</y>
    </hint>
    <hint type="destinationlabel">
     <x>556</x>
     <y>366</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_2</sender>
   <signal>clicked()</signal>
   <receiver>glaxnimate::gui::AboutEnvironmentDialog</receiver>
   <slot>open_settings_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_3</sender>
   <signal>clicked()</signal>
   <receiver>glaxnimate::gui::AboutEnvironmentDialog</receiver>
   <slot>open_backup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>653</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>glaxnimate::gui::AboutEnvironmentDialog</receiver>
   <slot>open_user_data()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>open_user_data()</slot>
  <slot>open_settings_file()</slot>
  <slot>copy_system()</slot>
  <slot>about_qt()</slot>
  <slot>dir_open(QModelIndex)</slot>
  <slot>open_backup()</slot>
 </slots>
</ui>
